server-common/src/main/java/org/apache/kafka/server/util/Utils.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.kafka.server.util;
+
+public class Utils {
+    public static void require(boolean requirement) {
```

Comment:
Any reason we are creating a new Utils in server instead of using the one in `server-common`?

Reply:
Sorry, can you make it clear. What existing class are you suggesting to use? 
I don't find right place to create `require` method :)

Reply:
I am referring to `org.apache.kafka.common.utils.Utils`

Reply:
Done